src/sdx_pce/topology/temanager.py

```diff
@@ -408,6 +408,11 @@ def get_links_on_path(self, solution: ConnectionSolution) -> list:
 
         return result
 
+    def add_breakdowns_to_connection(self, connection_request: dict, breakdowns: dict):
+        connection_request["breakdowns"] = breakdowns
+
+        return connection_request
```

Comment:
Can we add a docstring here?  The purpose of this method is not immediately obvious... I will likely wonder what this was about after a couple of months or weeks. :-)

Reply:
I agree-:). (1) the whole purpose is to associate the connect request with its computed breakdowns so sdx-controller can remember(2) I think this method might be absorbed by the pce.generate_connection_breakdown(), but I don't want to change any existing methods for . so this... when you work on unreserve_vlan workflow, feel free to change it if needed.